### DISCUSSION TITLE
모종의 이유로 마켓 or 랭킹 초기화를 못했을 시, 플레이어 내에서 초기화 할 수 있어야 함

### DISCUSSION THREAD

**Discussion by Namyujeong** [planetarium/NineChronicles#5880] [General]
https://planetariumhq.slack.com/archives/C062ZS68A21/p1709708885451109
현재는 앱 재실행을 해야만 다시 초기화를 할 수 있는 상황.
앱 내에서 새로고침이나 재시도가 가능한 환경이면 좋다.

**Comment by ipdae** [comment]
당장 이번 릴리즈에는 포함되기 힘들것같고 기능(또는 서비스)별로 나눠서 점진적으로 진행해야할것같습니다.
